src/screens/ProfileScreen/index.js

```diff
@@ -558,15 +557,8 @@ const ProfileScreen = ({route}) => {
     await downVote(post);
   };
 
-  const handleOnEndReached = () => {
-    if (!isLastPage) {
-      getMyFeeds(postOffset);
-    }
-  };
-
   function handleRefresh() {
     setLoading(true);
-    setIsLastPage(false);
     getMyFeeds(0, LIMIT_PROFILE_FEED);
     reloadFetchAnonymousPost();
     fetchMyProfile(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `handleOnEndReached` has been removed and replaced with `handleRefresh`. Make sure that the new implementation correctly handles the end of scrolling events and that the removal of `handleOnEndReached` does not affect other parts of your code that may rely on it.

```diff
- const handleOnEndReached = () => {
-   if (!isLastPage) {
-     getMyFeeds(postOffset);
-   }
- };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->